### DISCUSSION TITLE
Supporting TextAlign in widget.Entry

### DISCUSSION THREAD

**Discussion by hkparker** [fyne-io/fyne#4778] [Q&A] [answered]
This might be an uncommon one, but it would be nice to support text alignment inside of entries for my purposes (so that I can have the entered text centered inside the entry).  It appears that this isn't currently an option.

**Comment by andydotxyz** [comment]
Please don't use discussions to open a feature request.

**Comment by hkparker** [comment]
My apologies, I interpreted "ideas for new features" as being more or less feature requests on the start a discussion page.

![Screenshot from 2024-04-22 11-04-04](https://github.com/fyne-io/fyne/assets/2918615/ff90f065-949d-4128-a208-c47d1555ac2a)

Where would you like things like this?

**Reply by andydotxyz** [reply]
Apologies, that was GitHub defaults and we didn't update it - that is gone now.

You can see "New Feature Request" if you tap to create a new issue: https://github.com/fyne-io/fyne/issues/new/choose

**Reply by hkparker** [reply, marked as answer]
Got it, thanks!  Looking at issues it seems like there's already a conversation going on about RichText in entries.  I think that would cover alignment as well, so I'll refrain from opening a more specific request.
